### PR TITLE
dbplyr 2.3.0

### PR DESCRIPTION
**Pull request by hadley**
@mgirlich please let me know if there's anything else you think I should mention; and no rush, as I probably won't work on this again until early Jan.

### CODE REVIEW

File: content/blog/dbplyr-2-3-0/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+library(dplyr, warn.conflicts = FALSE)
+```
+
+## SQL optimisation
+
+dbplyr now produces fewer subqueries resulting in shorter, more readable, and,
+in some cases, faster SQL. Queries use `SELECT *` even after a join, where possible and the following combination of verbs now avoids subqueries much of the time:
+
+* `*_join()` + `select()` and `select()` + `*_join()`.
+* `mutate()` + `filter()` and `filter()` + `filter()` (@mgirlich, #792).
```
Comment (mgirlich) [inline content/blog/dbplyr-2-3-0/index.Rmd:55]:
```suggestion
* `mutate()` + `filter()` and `filter()` + `filter()`
```

File: content/blog/dbplyr-2-3-0/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+We're chuffed to announce the release of [dbplyr](http://dbplyr.tidyverse.org/) 2.3.0. dbplyr is a database backend for dplyr that allows you to use a remote database as if it was a collection of local data frames: you write ordinary dplyr code and dbplyr translates it to SQL for you.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("{package}")
+```
+
+This blog post will highlight some of the most important new features: eliminating subqueries for many verb combinations, better errors, and a handful of new translations.  As usual, release comes with a large number of improvements to translations for individual backends. See the full list in the [release notes]({ github_release })
```
Comment (mgirlich) [inline content/blog/dbplyr-2-3-0/index.Rmd:42]:
```suggestion
This blog post will highlight some of the most important new features: eliminating subqueries for many verb combinations, better errors, and a handful of new translations.  As usual, this release comes with a large number of improvements to translations for individual backends. See the full list in the [release notes]({ github_release })
```

**Comment by mgirlich** [comment]
For me the output is not on its own line (e.g. errors or SQL code)

image

I think this error was better before purrr 1.0.0

image

**Review by mgirlich — APPROVED** [review]

**Comment by mgirlich** [comment]
Very nice summary 👍

**Comment by hadley** [comment]
@mgirlich you mean if you preview it locally? If so, you need to install the latest hugodown.

**Comment by hadley** [comment]
To do:

* [ ] `use_tidy_thanks()`
* [ ] Update date

**Pull request merged**